<html> <head> <title>Folding@home</title></head><body>{{Infobox software
| name       = Folding@home
| screenshot =[[File:LifeWithPlayStation Folding.jpg|250px]]
| caption    = The [[PlayStation 3]] Folding@home client displays a [[3D computer graphics|3D model]] of the protein being simulated.
| author     = [[Vijay Pande]]
| developer  = [[Stanford University]] / Pande Group
| released   = 2000-10-01
| latest release version = ''Windows:''<br />6.23 (Uniprocessor)<br />6.32 (GPU)<br />''Mac OS X:''<br />6.24.1 (PPC-Uniprocessor)<br />6.29.3 (x86-SMP)<br />''Linux:''<br />6.02 (Uniprocessor)<br />6.29 (x64-SMP)<br />''PlayStation 3:'' 1.41 [http://www.scei.co.jp/folding/en/update.html]
| latest release date    = {{release date and age|2010|08|01}} (Windows GPU)
| latest preview version = ''Windows'':<br />6.30 beta (SMP)
| latest preview date    = {{release date and age|2010|07|16}} (SMP)
| operating system       = 
| platform   = [[Cross-platform]]
| language   = English
| genre      = [[Distributed computing]]
| license    = [[Proprietary software|Proprietary]] <ref name="f@hlicense">{{cite web |url=http://folding.stanford.edu/English/License |title=Folding@Home distributed computing client |author= |date= |work= |publisher=[[Stanford University]] |accessdate=26 August 2010}}</ref>
| website    = [http://folding.stanford.edu folding.stanford.edu]
}}

'''Folding@home ("Folding at Home")''' (sometimes abbreviated as '''FAH''' or '''F@h''') is a [[distributed computing]] (DC) project designed to perform computationally intensive simulations of [[protein folding]] and other [[molecular dynamics]] (MD), and to improve on the methods available to do so. It was launched on October 1, 2000, and is currently managed by the Pande Group, within [[Stanford University|Stanford University's]] [[chemistry]] department, under the supervision of Professor [[Vijay Pande]].

Folding@home is the most powerful distributed computing cluster in the world, according to [[Guinness World Records|Guinness]],<ref>[http://www.engadget.com/2007/10/31/folding-home-recognized-by-guinness-world-records/ Engadget, among other sites, announces that Guinness has recognized FAH as the most powerful distributed cluster, October 31, 2007. Retrieved November 5, 2007]</ref> and one of the world's largest distributed computing projects.<ref name="osstats">{{ cite web | url = http://fah-web.stanford.edu/cgi-bin/main.py?qtype=osstats | title = Client Statistics by OS | accessdate = 2008-01-05 |date= 2006-11-12 (updated automatically) | work = Folding@home distributed computing | publisher = Stanford University }}</ref> The goal of the project is "to understand protein folding, misfolding, and related diseases."<ref>{{ cite web | url = http://folding.stanford.edu | title = Folding@home distributed computing home page | accessdate = 2006-11-12 | author = Vijay Pande | year = 2006 | publisher = Stanford University }}</ref>

Accurate simulations of protein folding and misfolding enable the scientific community to better understand the development of many diseases, including [[sickle-cell disease]] (drepanocytosis), [[Alzheimer's disease]], [[Parkinson's disease]], [[Bovine spongiform encephalopathy]], [[cancer]], [[Huntington's disease]], [[cystic fibrosis]], [[osteogenesis imperfecta]], [[alpha 1-antitrypsin deficiency]], and other aggregation-related diseases.<ref>{{cite web
| url = http://folding.stanford.edu/FAQ-diseases.html
| title = Folding@home diseases studied FAQ
| publisher = Stanford University
}}</ref> More fundamentally, understanding the process of protein folding — how biological molecules assemble themselves into a functional state — is one of the outstanding problems of molecular biology. So far, the Folding@home project has successfully simulated folding in the 1.5 [[millisecond]] range<ref>{{cite web
| url = http://folding.typepad.com/news/2010/01/major-new-result-from-foldinghome-simulation-of-the-millisecond-timescale.html
| title = Folding@home: Paper #72: Major new result for Folding@home: Simulation of the millisecond timescale
}}</ref> — which is a simulation thousands of times longer than it was previously thought possible to model.

The Pande Group's goal is to refine and improve the MD and Folding@home DC methods to the level where it will become an essential tool for MD research,<ref>{{cite web
| url = http://twit.tv/fib27
| title = Futures in Biotech 27: Folding@home at 1.3 Petaflops
| format = Interview, webcast
}}</ref> and to achieve that goal they collaborate with various scientific institutions.<ref name="about">{{cite web
| url = http://folding.stanford.edu/English/About
| title = Folding@home - About
| format = FAQ
}}</ref> 
As of November 23, 2010, seventy-seven [[Academic publishing|scientific research papers]] have been published using the project's work.<ref name="papers">{{cite web
| url = http://folding.stanford.edu/English/Papers
| title = Folding@home - Papers
| accessdate = 2009-12-23
| author = Vijay Pande and the Folding@home team
| year = 2009
| work = Folding@home distributed computing
| publisher = Stanford University
}}</ref> A [[University of Illinois at Urbana-Champaign]] report dated October 22, 2002 states that Folding@home distributed simulations of [[protein folding]] are demonstrably accurate.<ref>{{cite journal
|author=C. Snow, H. Nguyen, V. S. Pande, and M. Gruebele.
|title=Absolute comparison of simulated and experimental protein-folding dynamics
|journal=Nature
|year=2002
|volume=420
|issue=6911
|pages= 102–106
|pmid=12422224|doi=10.1038/nature01160
}}</ref>

==Function==
[[Image:FAHMon.png|thumb|250px|Folding@home when running takes advantage of unused CPU cycles on a computer system as shown by this computer's 99% CPU usage.]]
Folding@home does not rely on powerful [[supercomputer]]s for its [[data processing]]; instead, the primary contributors to the Folding@home project are many hundreds of thousands of [[personal computer]] users who have installed a [[client (computing)|client]] [[computer program|program]]. The client runs in the [[background (computer software)|background]], utilizing otherwise unused [[central processing unit|CPU]] (or in some cases, [[Graphical Processing Unit|GPU]], see below) power. The older, no longer used version of Folding@home for the CPU could be run as a screen saver, only folding while the user is away. In most modern personal computers, the CPU is rarely used to its full capacity at all times; the Folding@home client takes advantage of this unused processing power.

The Folding@home client periodically connects to a [[Server (computing)|server]] to retrieve "work units", which are packets of data upon which to perform calculations. Each completed work unit is then sent back to the server. As data integrity is a major concern for all distributed computing projects, all work units are validated through the use of a 2048 bit [[digital signature]].

Contributors to Folding@home may have [[user name]]s used to keep track of their contributions. Each user may be running the client on one or more CPUs; for example, a user with two computers could run the client on both of them. Users may also contribute under one or more team names; many different users may join together to form a team. Contributors are assigned a score indicating the number and difficulty of completed work units. Rankings and other statistics are posted to the Folding@home website http://folding.stanford.edu/.

== Software ==

The Folding@home client consists of three separate components.
*The client software acts as a download and file manager for work units and scientific cores, controls the cores, and is the software with which the user interacts. Separating the client from the core enables the scientific methods to be updated automatically (or new methods to be added) without a client update.
*The Work Unit is the actual data that the client is being asked to process.
*The Core performs the calculations on the work unit. Folding@home's cores are based on modified versions of seven [[Molecular dynamics|molecular simulation]] programs for calculation: [[TINKER]], [[GROMACS]], [[AMBER]], [[CPMD]], [[SHARPEN]], [[ProtoMol]] and [[Desmond (software)|Desmond]].<ref>{{cite web
| url = http://folding.stanford.edu/QMD.html
| title = Folding@home with QMD core FAQ
| accessdate = 2006-12-03
| author = Vijay Pande
| date= 2005-10-16
| format = FAQ| publisher = Stanford University
}} ''The site indicates that Folding@home uses a modification of CPMD allowing it to run on the supercluster environment.''</ref><ref name="fahdev">{{cite web
| url = http://folding.typepad.com/news/2009/06/how-does-fah-code-development-and-sysadmin-get-done.html
| title = Folding@home: How does FAH code development and sysadmin get done?
| accessdate = 2009-06-25
| author = Vijay Pande
| date = 2009-06-17}}</ref> Where possible, [[Compiler optimization|optimizations]] are used to speed the process of calculation. There are many variations on these base simulation programs, each of which is given an [[Arbitrariness|arbitrary]] identifier (Core xx):<ref name="cores">{{cite web
| url = http://fahwiki.net/index.php/Cores
| title = Cores - FaHWiki
| accessdate = 2007-11-06
| format = FAQ
}}</ref>
:

===Active Cores===
Cores listed in this section may not necessarily be being used by the project at any given time, but are included here as their use may be resumed at any time without notice. Deprecated and forthcoming cores are in the next section.
* [[GROMACS]] (all variants of this core use [[SIMD]] optimizations including [[Streaming SIMD Extensions|SSE]], [[3dnow|3DNow+]] or [[AltiVec]], where available, unless otherwise specified)
** Gromacs (Core 78)
*** Available for all [[Uniprocessor system|Uniprocessor]] clients only.
** DGromacs (Core 79)
*** [[Double precision]] Gromacs, uses [[SSE2]] only.
*** Available for all Uniprocessor clients only.
** DGromacsB (Core 7b)
*** Nominally an update of DGromacs, but is actually based on the [[Symmetric multiprocessing|SMP]]/[[Graphics processing unit|GPU]] codebases (and is therefore a completely new core). As a result, both are still in use.
*** Double precision Gromacs, uses SSE2 only.
*** Available for all Uniprocessor clients only.
** DGromacsC (Core 7c)
*** Double precision Gromacs, uses SSE2 only.
*** Available on [[Microsoft Windows|Windows]] and [[Linux]] Uniprocessor clients only. 
** GBGromacs (Core 7a)
*** Gromacs with the [[Generalized Born]] [[Implicit solvation|implicit solvent]] model.
*** Available for all Uniprocessor clients only.
** Gromacs SREM (Core 80)
*** Gromacs Serial [[Parallel tempering|Replica Exchange]] Method.
*** The Gromacs Serial Replica Exchange Method core, also known as GroST (Gromacs Serial replica exchange with Temperatures), uses the Replica Exchange method (also known as REMD or Replica Exchange Molecular Dynamics) in its simulations.
*** Available for Windows and Linux Uniprocessor clients only.
** GroSimT (Core 81)
*** Gromacs with Simulated Tempering.
*** Available for Windows and Linux Uniprocessor clients only.
** Gromacs 33 (Core a0)
*** Uses the Gromacs 3.3 [[codebase]].
*** Available for all Uniprocessor clients only. [[Image:GPU - F@H.jpg|thumb|right| NVIDIA GPU v2.0 r1 client for Windows.]]
** Gro-A3 core (Core a3)
*** SMP version of the Gromacs A4 core.
*** Uses threads rather than MPI for multicore support.
*** Available for SMP2 client only.
*** In open beta testing before general release.
*** Released January 24, 2010.
** Gro-A4 (Core a4)
*** A single core version of the Gromacs SMP2 core.
*** Available for Windows and Linux Uniprocessor clients only.
** GroGPU2 (Core 11)
*** [[Graphics processing unit|Graphics Processing Unit]] variant for [[ATI Technologies|ATI]] [[Compute Abstraction Layer|CAL]]-enabled and [[Nvidia|nVidia]] [[CUDA]]-enabled GPUs.
*** Comes in two separate versions, one each for ATI and nVidia, but both have the same Core ID.
*** GPUs do not support SIMD optimizations by design, so none are used in this core.
*** Available for GPU2 client only.
** ATI-DEV (Core 12)
*** Graphics Processing Unit developmental core for ATI CAL-enabled GPUS.
*** Does not support SIMD optimizations.
*** Available for GPU2 client only.
** NVIDIA-DEV (Core 13)
*** Graphics Processing Unit developmental core for nVidia CUDA-enabled GPUs.
*** Does not support SIMD optimizations.
*** Available for GPU2 client only.
** GroGPU2-MT (Core 14)<ref>{{cite web
| url = http://foldingforum.org/viewtopic.php?f=52&t=8734&start=0
| title = Folding Forum: Announcing project 5900 and Core_14 on advmethods
| accessdate = 2009-03-02
| author = 
| year = 2009
| publisher = 
}}</ref>
*** Graphics Processing Unit variant for nVidia CUDA-enabled GPUs.
*** Contains additional debugging code compared to the standard Core 11.
*** Does not support SIMD optimizations.
*** Released March 2, 2009.
*** Available for GPU2 client only.
** Gro-PS3 (Does not have a known ID number, but also called SCEARD core)
*** [[PlayStation 3]] variant.
*** No SIMD optimizations, uses [[Cell (microprocessor)#Synergistic Processing Elements (SPE)|SPE]] cores for optimization.
*** Available for PS3 client only.
* [[AMBER]]
** PMD (Core 82)<ref name="cores"/>
*** No optimizations.
*** Available for Windows and Linux Uniprocessor clients only.
* [[ProtoMol]] <ref name="about"/>
** Protomol Core (Core b4)
***In open beta testing before general release.
***Released to open testing on February 11, 2010.
* [[OpenMM]]
** Gromacs-OpenMM Core (Core 15)
***In open beta testing before general release.
***Uses CUDA and is available for nVidia only.
***Available for GPU2 client only.

===Inactive Cores===
* [[TINKER]]
** Tinker core (Core 65)
*** Retired, as the GBGromacs cores (Cores 7a and a4) perform the same tasks much faster.
*** No optimizations.
*** Available for all Uniprocessor clients only.
* GROMACS
** GroGPU (Core 10)
*** Graphics Processing Unit variant for [[ATI Technologies|ATI]] [[Radeon R520|series 1xxx]] [[Graphics Processing Unit|GPUs]].
*** GPUs do not have optimizations; no SIMD optimizations needed since GPU cores are explicitly designed for SIMD.
*** Retired as of June 6, 2008 due to end of distribution of GPU1 client units.
*** Available for GPU1 client only.
** Gro-SMP (Core a1)
*** [[Symmetric multiprocessing|Symmetric MultiProcessing]] variant, locked to four [[Thread (computer science)|threads]] (but can be run on [[Multi-core|dual core]] processors).
*** Runs only on [[multi-core]] [[x86]] or [[x86-64|x64]] hardware.
*** Was available for all SMP1 clients only.
*** Retired due to move to threads-based SMP with the SMP2 client.
** GroCVS (Core a2)
*** Symmetric MultiProcessing variant with scalable numbers of threads.
*** Runs only on [[multi-core]] [[x86]] or [[x86-64|x64]] hardware, with four or more cores.
*** Uses the Gromacs 4.0 codebase.
*** Was available for Linux and Mac OS X SMP1 clients only.
*** Retired due to move to threads-based SMP with the SMP2 client.
* [[CPMD]]
** QMD (Core 96)
*** Currently inactive, due to QMD developer graduating from [[Stanford University]] and due to current research shifting away from Quantum MD. 
*** Caused controversy due to SSE2 issues involving [[Intel Corporation|Intel]] libraries and [[Advanced Micro Devices|AMD]] processors.<ref>{{cite web
| url = http://fahwiki.net/index.php/FAH_&_QMD_&_AMD64_&_SSE2
| title = FAH & QMD & AMD64 & SSE2
| format = FAQ
}}</ref>
*** Uses SSE2 (currently only on Intel CPUs, see above).
*** Available for Windows and Linux Uniprocessor clients only.
* [[SHARPEN]] <ref>{{cite web
| url = http://p450.caltech.edu/sharpen/sharpenabout.html
| title = SHARPEN: Systematic Hierarchical Algorithms for Rotamers and Proteins on an Extended Network
| format = About
}}</ref>
**SHARPEN Core <ref>{{cite web
| title = SHARPEN
| url = http://p450.caltech.edu/sharpen/sharpenprojects.html
}}</ref>
***Currently inactive, in closed [[Software release life cycle#Beta|beta testing]] before [[Software release life cycle#GA|general release]].
***Uses different format to standard F@H cores, as there is more than one "Work Unit" (using the normal definition) in each work packet sent to clients.
* [[Desmond (software)|Desmond]]
**Desmond Core
***Currently inactive, in closed beta testing before general release.
***Will be available for Uniprocessor and SMP2 clients.
* OpenMM
**OpenMM-Gromacs core (Core 16)
***An updated version of Core 15, using OpenCL to support both ATI and nVidia graphics cards.
***Currently inactive, in closed beta testing before general release.
***Will be available for GPU3 client only.

== Participation ==
[[Image:FAH-tflops.PNG|thumb|right|260px|Folding@home computing power shown - by device type - in [[TeraFLOPS]] as recorded semi-daily from November 2006 until September 2007. Note the large spike in total compute power after March 22, when the [[PlayStation 3]] client was released.]]
Shortly after breaking the 200,000 active CPU count on September 20, 2005, the Folding@home project celebrated its fifth anniversary on October 1, 2005.

Interest and participation in the project has grown steadily since its launch. The number of active devices participating in the project increased substantially after receiving much publicity during the launch of their high performance clients for both [[ATi]] [[Video Card|graphics cards]] and the PlayStation 3, and again following the launch of the high performance client for [[Nvidia|nVidia]] graphics cards.

As of April 9, 2009 the peak speed of the project overall has reached over 5.0 native [[petaflop|PFLOPS]] (8.1 x86 PFLOPS<ref name="Folding@home FLOP FAQ">[http://folding.stanford.edu/English/FAQ-flops Folding@home FLOP FAQ]</ref>) from around 400,000 active machines, and the project has received computational results from over 4.51 million devices since it first started.<ref name="osstats"/>

===Google & Folding@home===
There used to be cooperation between Folding@home and [[Google]] Labs in the form of [[Google Compute|Google Toolbar]]. Google Compute supported Folding@home during its early stage — when Folding@home had about 10,000 active CPUs. At that time, a boost of 20,000 machines was very significant. Today the project has a large number of active CPUs and the number of new clients joining Google Compute was very low (most people opted for the Folding@home client instead), it is available [http://toolbar.google.com/dc/aboutdc.html here] (must access in IE with old version of Google Toolbar). The Google Compute clients also had certain limits: they could only run the TINKER core and had limited naming  and team options. Folding@home is no longer supported on Google Toolbar, and even the old Google Toolbar client will not work.<ref>{{cite web
| url = http://forum.folding-community.org/fpost151025.html
| title = What is the state of Google Compute client?
| accessdate = 2006-11-12
| format = Blog
| work = Folding@home support forum
| publisher = Stanford University
}}</ref>

===Genome@home===

Folding@home absorbed the [[Genome@home]] project on March 8, 2004. The work which was started by the Genome@home project has since been completed using the Folding@home network (the work units without deadlines), and no new work is being distributed by this project. All donators were encouraged to download the Folding@home client (the F@h 4.xx client had a Genome@home option), and once the Genome@home work was complete these clients were asked to donate their processing power to the Folding@home project instead.

===PetaFLOPS milestones===
{| align="right" border="1" class="wikitable" style="margin-left:5px"
|-
! Native petaFLOPS Barrier
! Date Crossed
|-
| align="center" | 1.0
| September 16, 2007
|-
| align="center" | 2.0
| early May 2008
|-
| align="center" | 3.0
| August 20, 2008
|-
| align="center" | 4.0
| September 28, 2008
|-
| align="center" | 5.0
| February 18, 2009
|}
On September 16, 2007, the Folding@home project officially attained a sustained performance level higher than one native [[FLOPS|petaFLOPS]], becoming the first computing system of any kind in the world to ever do so, although it had briefly peaked above one native petaFLOPS in March 2007, receiving a large amount of mainstream media coverage for doing so.<ref>[http://folding.typepad.com/news/2007/09/crossing-the-pe.html Folding@home: Crossing the petaFLOPS barrier<!-- Bot generated title -->]</ref><ref>[http://folding.typepad.com/news/2007/09/post-petaflop.html Folding@home: Post petaflop<!-- Bot generated title -->]</ref> In early May 2008 the project attained a sustained performance level higher than two native petaFLOPS, followed by the three and four native petaFLOPS milestones on August 20 and September 28, 2008 respectively. On February 18, 2009, Folding@home achieved a performance level of just above 5 [[FLOPS|petaFLOPS]], thereby becoming the first computing system of any kind to surpass 5 native PFLOPS,<ref>[http://folding.typepad.com/news/2009/02/foldinghome-passes-the-5-petaflop-mark.html "Folding@home passes the 5 petaflop mark"] from the official Folding@home blog</ref> just as it was for the other four milestones.

The Folding@home computing cluster currently operates at above 5.6 native petaFLOPS, with a large majority of the performance coming from [[GPU]] and [[PlayStation 3]] clients.<ref name="osstats"/> In comparison to this, the fastest standalone [[supercomputer]] (non-distributive computing) in the world (as of November 2010, [[Tianhe-I]]) peaks at approximately 2.56 petaFLOPS.<ref>
{{cite web
|url=http://www.top500.org/system/performance/10587
|title=TOP500 Tianhe-1A Performance Data 
|accessdate=2010-11-29
}}</ref>

Beginning in April 2009, Folding@Home began reporting performance in both "Native" FLOPS and x86 FLOPS.<ref name="osstats"/> ("x86" FLOPS reported at a much higher mark than the "Native" FLOPS) A detailed explanation of the difference between the two figures was given in the FLOP section of the Folding@Home FAQ.<ref name="Folding@home FLOP FAQ"/>

== Results ==
These peer-reviewed papers (in chronological order) all use research from the Folding@home project.<ref name="papers"/>

===2000–2001===

*{{cite journal
|author=M. R. Shirts and V. S. Pande.
|title=Screen Savers of the World, Unite!
|journal=Science
|year=2000
|volume=290
|issue=5498
|pages= 1903–1904
|doi=10.1126/science.290.5498.1903
|pmid=17742054
}}
*{{cite journal
|author=Michael R. Shirts and Vijay S. Pande
|title=Mathematical Analysis of Coupled Parallel Simulations
|journal=Physical Review Letters
|year=2001
|volume=86
|issue=22
|pages=4983–4987
|doi=10.1103/PhysRevLett.86.4983
|pmid=11384401
}}
*{{cite journal
|author=Bojan Zagrovic, Eric J. Sorin and Vijay Pande
|title=b-Hairpin Folding Simulations in Atomistic Detail Using an Implicit Solvent Model
|journal=Journal of Molecular Biology
|year=2001
|volume=313
|issue=1
|pages=151–169
|doi=10.1006/jmbi.2001.5033
|pmid=11601853
}}

===2002===

*Stefan M. Larson, Christopher D. Snow, Michael R. Shirts, and Vijay S. Pande (2002) "Folding@home and Genome@home: Using distributed computing to tackle previously intractable problems in computational biology", Stefan M. Larson, Christopher D. Snow, Michael R. Shirts, and Vijay S. Pande. Published in ''Computational Genomics'', Richard Grant (2004), Horizon Press 
*{{cite journal
|author=Bojan Zagrovic, Christopher D. Snow, Michael R. Shirts, and Vijay S. Pande.
|title=Simulation of Folding of a Small Alpha-helical Protein in Atomistic Detail using Worldwide distributed Computing
|journal=Journal of Molecular Biology
|year=2002
|volume=323
|issue=5
|pages=927–937
|doi=10.1016/S0022-2836(02)00997-X
|pmid=12417204}}
*{{cite journal
|author=Bojan Zagrovic, Christopher D. Snow, Siraj Khaliq, Michael R. Shirts, and Vijay S. Pande
|title=Native-like Mean Structure in the Unfolded Ensemble of Small Proteins
|journal=Journal of Molecular Biology
|year=2002
|volume=323
|issue=1
|pages=153–164
|doi=10.1016/S0022-2836(02)00888-4
|pmid=12368107}}
*{{cite journal
|author=Christopher D. Snow, Bojan Zagrovic, and Vijay S. Pande
|title=The Trp Cage: Folding Kinetics and Unfolded State Topology via Molecular Dynamics Simulations
|journal=Journal of the American Chemical Society
|year=2002
|volume=124
|issue=49
|pages=14548–14549
|doi=10.1021/ja028604l
|pmid=12465960}}

===2003===

*{{cite journal
|author=Vijay S. Pande, Ian Baker, Jarrod Chapman, Sidney P. Elmer, Siraj Khaliq, Stefan M. Larson, Young Min Rhee, Michael R. Shirts, Christopher D. Snow, Eric J. Sorin, Bojan Zagrovic
|title=Atomistic protein folding simulations on the submillisecond timescale using worldwide distributed computing
|journal=Biopolymers
|year=2003
|volume=68
|issue=1
|pages=91–109
|doi=10.1002/bip.10219
|pmid=12579582
}}
*{{cite journal
|author=Young Min Rhee & Vijay S. Pande
|title=Multiplexed-Replica Exchange Molecular Dynamics Method for Protein Folding Simulation
|journal=Biophysical Journal
|year=2003
|volume=84
|issue=2
|pages=775–786
|doi=10.1016/S0006-3495(03)74897-8
|pmid=12547762
|pmc=1302658
}}
*{{cite journal
|author=Eric J. Sorin, Young Min Rhee, Bradley J. Nakatani & Vijay S. Pande
|title=Insights Into Nucleic Acid Conformational Dynamics from Massively Parallel Stochastic Simulations
|journal=Biophysical Journal
|year=2003
|volume=85
|issue=2
|pages=790–803
|doi=10.1016/S0006-3495(03)74520-2
|pmid=12885628
|pmc=1303202
}}
*{{cite journal
|author=Bojan Zagrovic and Vijay S. Pande
|title=Solvent Viscosity Dependence of the Folding Rate of a Small Protein: Distributed Computing Study
|journal=Journal of Computational Chemistry
|year=2003
|volume=24
|issue=12
|pages=1432–1436
|doi=10.1002/jcc.10297
|pmid=12868108
}}
*{{cite journal
|author=Michael R. Shirts, Jed W. Pitera, William C. Swope, and Vijay S. Pande
|title=Extremely precise free energy calculations of amino acid side chain analogs: Comparison of common molecular mechanics force fields for proteins
|journal=Journal of Chemical Physics
|year=2003
|volume=119
|issue=11
|pages=5740–5761
|doi=10.1063/1.1587119
}}
*{{cite journal
|author=Michael R. Shirts, Eric Bair, Giles Hooker, and Vijay S Pande
|title=Equilibrium Free Energies from Nonequilibrium Measurements Using Maximum-Likelihood Methods
|journal=Physical Review Letters
|year=2003
|volume=91
|issue=14
|pmid=14611511
|pages=140601
|doi=10.1103/PhysRevLett.91.140601
}}
*{{cite journal
|author=Bojan Zagrovic & Vijay S Pande
|title=Structural correspondence between the alpha-helix and the random-flight chain resolves how unfolded proteins can have native-like properties
|journal=Nature Structural Biology
|year=2003
|volume=10
|issue=11
|pages=955–961
|doi=10.1038/nsb995
|pmid=14555998
}}

===2004===

*{{cite journal
|author=Eric J. Sorin, Bradley J. Nakatani, Young Min Rhee, Guha Jayachandran, V Vishal, & Vijay S Pande
|title=Does Native State Topology Determine the RNA Folding Mechanism?
|journal=Journal of Molecular Biology
|year=2004
|volume=337
|issue=4
|pages=789–757
|doi=10.1016/j.jmb.2004.02.024
|pmid=15033351}}
*{{cite journal
|author=Christopher D. Snow, Linlin Qiu, Deguo Du, Feng Gai, Stephen J. Hagen, & Vijay S Pande
|title=Trp zipper folding kinetics by molecular dynamics and temperature-jump spectroscopy
|journal=Proceedings of the [[United States National Academy of Sciences|National Academy of Sciences, USA]]
|year=2004
|volume=101
|issue=12
|pages=4077–4082
|doi=10.1073/pnas.0305260101
|pmid=15020773
|pmc=384698}}
*{{cite journal
|author=Young Min Rhee, Eric J. Sorin, Guha Jayachandran, Erik Lindahl, & Vijay S Pande
|title=Simulations of the role of water in the protein-folding mechanism
|journal=Proceedings of the National Academy of Sciences, USA
|year=2004
|volume=101
|issue=17
|pages=6456–6461
|doi=10.1073/pnas.0307898101
|pmid=15090647
|pmc=404066}}
*{{cite journal
|author=Nina Singhal, Christopher D. Snow, and Vijay S. Pande
|title=Using path sampling to build better Markovian state models: Predicting the folding rate and mechanism of a tryptophan zipper beta hairpin
|journal=Journal of Chemical Physics
|year=2004
|volume=121
|issue=1
|pages=415–425
|doi=10.1063/1.1738647
|pmid=15260562}}
*{{cite journal
|author=L. T. Chong, C. D. Snow, Y. M. Rhee, and V. S. Pande.
|title=Dimerization of the p53 Oligomerization Domain: Identification of a Folding Nucleus by Molecular Dynamics Simulations
|journal=Journal of Molecular Biology
|year=2004
|volume=345
|issue=4
|pages= 869–878
|doi=10.1016/j.jmb.2004.10.083
|pmid=15588832}}

===2005===

*{{cite journal
|author=Eric J. Sorin, Young Min Rhee, and Vijay S. Pande
|title=Does Water Play a Structural Role in the Folding of Small Nucleic Acids?
|journal=Biophysical Journal
|year=2005
|volume=88
|issue=4
|pages=2516–2524
|doi=10.1529/biophysj.104.055087
|pmid=15681648
|pmc=1305348}}
*{{cite journal
|author=Eric J. Sorin and Vijay S. Pande
|title=Exploring the Helix-Coil Transition via All-atom Equilibrium Ensemble Simulations
|journal=Biophysical Journal
|year=2005
|volume=88
|issue=4
|pages=2472–2493
|doi=10.1529/biophysj.104.051938
|pmid=15665128
|pmc=1305346}}
*{{cite journal
|author=Eric J. Sorin and Vijay S. Pande
|title=Empirical Force-Field Assessment: The Interplay Between Backbone Torsions and Noncovalent Term Scaling
|journal=Journal of Computational Chemistry
|year=2005
|volume=26
|issue=7
|pages=682–690
|doi=10.1002/jcc.20208
|pmid=15754305}}
*{{cite journal
|author=C. D. Snow, E. J. Sorin, Y. M. Rhee, and V. S. Pande.
|title=How well can simulation predict protein folding kinetics and thermodynamics?
|journal=Annual Reviews of Biophysics
|year=2005
|volume=34
|issue=
|pages=43–69
|doi=10.1146/annurev.biophys.34.040204.144447
|pmid=15869383}}
*{{cite journal
|author=Bojan Zagrovic, Jan Lipfert, Eric J. Sorin, Ian S. Millett, Wilfred F. van Gunsteren, Sebastian Doniach & Vijay S. Pande
|title=Unusual compactness of a polyproline type II structure
|journal=Proceedings of the National Academy of Sciences, USA
|year=2005
|volume=102
|issue=33
|pages=11698–11703
|doi=10.1073/pnas.0409693102
|pmid=16085707
|pmc=1187952}}
*{{cite journal
|author=Michael R. Shirts & Vijay S. Pande
|title=Comparison of efficiency and bias of free energies computed by exponential averaging, the Bennett acceptance ratio, and thermodynamic integration
|journal=Journal of Chemical Physics
|year=2005
|volume=122
|issue=14
|pages=144107
|doi=10.1063/1.1873592
|pmid=15847516}}
*{{cite journal
|author=Michael R. Shirts & Vijay S. Pande
|title=Solvation free energies of amino acid side chain analogs for common molecular mechanics water models
|journal=Journal of Chemical Physics
|year=2005
|volume=122
|issue=13
|pages=134508
|doi=10.1063/1.1877132
|pmid=15847482}}
*{{cite journal
|author=Sidney Elmer, Sanghyun Park, & Vijay S. Pande
|title=Foldamer dynamics expressed via Markov state models. I. Explicit solvent molecular-dynamics simulations in acetonitrile, chloroform, methanol, and water
|journal=Journal of Chemical Physics
|year=2005
|volume=123
|issue=
11
|pmid=16392592|pages=114902
|doi=10.1063/1.2001648}}
*{{cite journal
|author=Sidney Elmer, Sanghyun Park, & Vijay S. Pande
|title=Foldamer dynamics expressed via Markov state models. II. State space decomposition
|journal=Journal of Chemical Physics
|year=2005
|volume=123
|issue=
11
|pmid=16392593|pages=114903
|doi=10.1063/1.2008230}}
*{{cite journal
|author=Sanghyun Park, Randall J. Radmer, Teri E. Klein, and Vijay S. Pande
|title=A New Set of Molecular Mechanics Parameters for Hydroxyproline and Its Use in Molecular Dynamics Simulations of Collagen-Like Peptides
|journal=Journal of Computational Chemistry
|year=2005
|volume=26
|issue=15
|pages=1612–1616
|doi=10.1002/jcc.20301
|pmid=16170799}}
*{{cite journal
|author=Hideaki Fujutani, Yoshiaki Tanida, Masakatsu Ito, Guha Jayachandran, Christopher D. Snow, Michael R. Shirts, Eric J. Sorin, and Vijay S. Pande
|title=Direct calculation of the binding free energies of FKBP ligands using the Fujitsu BioServer massively parallel computer
|journal=Journal of Chemical Physics
|year=2005
|volume=123
|issue=8
|pages=084108
|doi=10.1063/1.1999637
|pmid=16164283}}
*{{cite journal
|author=Nina Singhal and Vijay S. Pande
|title=Error Analysis and efficient sampling in Markovian State Models for protein folding
|journal=Journal of Chemical Physics
|year=2005
|volume=123
|issue=20
|pages=204909
|doi=10.1063/1.2116947
|pmid=16351319}}
*{{cite journal
|author=Bojan Zagrovic, Guha Jayachandran, Ian S. Millett, Sebastian Doniach and Vijay S. Pande
|title=How large is alpha-helix in solution? Studies of the radii of gyration of helical peptides by SAXS and MD
|journal=Journal of Chemical Physics
|year=2005
|volume=353
|issue=2
|pages=232–241
|doi=10.1016/j.jmb.2005.08.053
|pmid=16171817}}

===2006===

*{{cite journal
|author=Paula Petrone and Vijay S. Pande
|title=Can conformational change be described by only a few normal modes?
|journal=Biophysical Journal
|year=2006
|volume=90
|issue=5
|pages=1583–1593
|doi=10.1529/biophysj.105.070045
|pmid=16361336
|pmc=1367309}}
*{{cite journal
|author=Eric J. Sorin, Young Min Rhee, Michael R. Shirts, and Vijay S. Pande
|title=The solvation interface is a determining factor in peptide conformational preferences
|journal=Journal of Molecular Biology
|year=2006
|volume=356
|issue=1
|pages=248–256
|doi=10.1016/j.jmb.2005.11.058
|pmid=16364361}}
*{{cite journal
|author=Eric J. Sorin and Vijay S. Pande
|title=Nanotube confinement denatures protein helices
|journal=Journal of the American Chemical Society
|year=2006
|volume=128
|issue=19
|pages=6316–6317
|doi=10.1021/ja060917j
|pmid=16683786}}
*{{cite journal
|author=Young Min Rhee and Vijay S. Pande
|title=On the role of chemical detail in simulating protein folding kinetics
|journal=Chemical Physics
|year=2006
|volume=323
|issue=
|pages=66–77
|doi=10.1016/j.chemphys.2005.08.060}}
*{{cite journal
|author=L.T. Chong, W. C. Swope, J. W. Pitera, and V. S. Pande
|title=A novel approach for computational alanine scanning: application to the p53 oligomerization domain
|journal=Journal of Molecular Biology
|year=2006
|volume=357
|issue=3
|pages=1039–1049
|doi=10.1016/j.jmb.2005.12.083
|pmid=16457841}}
*{{cite journal
|author=I. Suydam, C. D. Snow, V. S. Pande and S. G. Boxer.
|title=Electric Fields at the Active Site of an Enzyme: Direct Comparison of Experiment with Theory
|journal=Science
|year=2006
|volume=313
|issue=5784
|pages=200–204
|doi=10.1126/science.1127159
|pmid=16840693}}
*{{cite journal
|author=P. Kasson, N. Kelley, N. Singhal, M. Vrjlic, A. Brunger, and V. S. Pande
|title=Ensemble molecular dynamics yields submillisecond kinetics and intermediates of membrane fusion
|journal=Proceedings of the National Academy of Sciences, USA
|year=2006
|volume=103
|issue=32
|pages=11916–11921
|doi=10.1073/pnas.0601597103
|pmid=16880392
|pmc=1567673}}
*{{cite journal
|author=Guha Jayachandran, V. Vishal, and V. S. Pande
|title=Folding Simulations of the Villin Headpiece in All-Atom Detail
|journal=Journal of Chemical Physics
|year=2006
|volume=124
|issue=16
|pages=164902
|doi=10.1063/1.2186317
|pmid=16674165}}
*{{cite journal
|author=Guha Jayachandran, M. R. Shirts, S. Park, and V. S. Pande
|title=Parallelized Over Parts Computation of Absolute Binding Free Energy with Docking and Molecular Dynamics
|journal=Journal of Chemical Physics
|year=2006
|volume=125
|issue=8
|pages=084901
|doi=10.1063/1.2221680
|pmid=16965051}}
*{{cite journal
|author=C. Snow and V. S. Pande
|title=Kinetic Definition of Protein Folding Transition State Ensembles and Reaction Coordinates
|journal=Biophysical Journal
|year=2006
|volume=91
|issue=1
|pages=14–24
|doi=10.1529/biophysj.105.075689
|pmid=16617068
|pmc=1479057}}
*{{cite journal
|author=S. Park, D. L. Ensign, and V. S. Pande
|title=A Bayesian Update Method for Adaptive Weighted Sampling
|journal=Physical Review E
|year=2006
|volume=74
|issue=6
|pages=066703
|doi=10.1103/PhysRevE.74.066703}}
*{{cite journal
|author=P. Kasson and V. S. Pande
|title=Predicting structure and dynamics of loosely-ordered protein complexes: influenza hemagglutinin fusion peptide
|journal=PSB
|year=2006
|volume=
|issue=
|pages=40–50
|doi=10.1142/9789812772435_0005
|pmid=17992744}}
*{{cite journal
|author=Erich Elsen, Mike Houston, V. Vishal, Eric Darve, Pat Hanrahan, and Vijay Pande
|title=N-Body simulation on GPUs
|journal=Proceedings of the 2006 [[Association for Computing Machinery|ACM]]/[[Institute of Electrical and Electronics Engineers|IEEE]] conference on Supercomputing
|year=2006
|volume=
|issue=
|pages=188
|doi=10.1145/1188455.1188649}}

===2007===

*{{cite journal
|author=Guha Jayachandran, V. Vishal, Angel E. Garcıa and V. S. Pande
|title=Local structure formation in simulations of two small proteins
|journal=Journal of Structural Biology
|year=2007
|volume=157
|issue=3
|pages=491–499
|doi=10.1016/j.jsb.2006.10.001
|pmid=17098444
|pmc=1931413}}
*{{cite journal
|author=Adam L Beberg and Vijay S. Pande
|title=[[Storage@home]]: Petascale Distributed Storage
|journal=IPDPS
|year=2007
|volume=
|issue=
|pages=1
|doi=10.1109/IPDPS.2007.370672}}
*{{cite journal
|author=J. Chodera, N. Singhal, V. S. Pande, K. Dill, and W. Swope
|title=Automatic discovery of metastable states for the construction of Markov models of macromolecular conformational dynamics
|journal=Journal of Chemical Physics
|year=2007
|volume=126
|issue=15
|pages=155101
|doi=10.1063/1.2714538
|pmid=17461665}}
*{{cite journal
|author=D. Lucent, V. Vishal, V. S. Pande
|title=Protein folding under confinement: a role for solvent
|journal=Proceedings of the National Academy of Sciences, USA
|year=2007
|volume=104
|issue=25
|pages=10430–10434
|doi=10.1073/pnas.0608256104
|pmid=17563390
|pmc=1965530}}
*{{cite journal
|author=P. M. Kasson, A. Zomorodian, S. Park, N. Singhal, L. J. Guibas, and V. S. Pande
|title=Persistent voids: a new structural metric for membrane fusion
|journal=Bioinformatics
|year=2007
|volume=23
|issue=14
|pages=1753–9
|doi=10.1093/bioinformatics/btm250
|pmid=17488753}}
*{{cite journal
|author=P. M. Kasson and V. S. Pande
|title=Control of Membrane Fusion Mechanism by Lipid Composition: Predictions from Ensemble Molecular Dynamics
|journal=PLoS Computational Biology
|year=2007
|volume=3
|issue=11
|pages=e220
|doi=10.1371/journal.pcbi.0030220
|pmid=18020701
|pmc=2077900}}
*{{cite journal
|author=D. L. Ensign, P. M. Kasson, and V. S. Pande
|title=Heterogeneity Even at the Speed Limit of Folding: Large-scale Molecular Dynamics Study of a Fast-folding Variant of the Villin Headpiece
|journal=Journal of Molecular Biology
|year=2007
|volume=374
|issue=3
|pages=806–816
|doi=10.1016/j.jmb.2007.09.069
|pmid=17950314}}
*{{cite journal
|author=Alex Robertson, Edgar Luttmann, Vijay S. Pande
|title=Effects of long-range electrostatic forces on simulated protein folding kinetics
|journal=Journal of Computational Chemistry
|year=2007
|volume=29
|issue=5
|pages=694–700
|doi=10.1002/jcc.20828
|pmid=17849394}}
*{{cite journal
|author=Nina Singhal Hinrichs and Vijay S. Pande
|title=Calculation of the distribution of eigenvalues and eigenvectors in Markovian state models for molecular dynamics
|journal=Journal of Chemical Physics
|year=2007
|volume=126
|issue=24
|pages=244101
|doi=10.1063/1.2740261
|pmid=17614531}}

===2008===

*{{cite journal
|author=Xuhui Huang, Gregory R. Bowman,and Vijay S. Pande
|title=Convergence of folding free energy landscapes via application of enhanced sampling methods in a distributed computing environment
|journal=Journal of Chemical Physics
|year=2008
|volume=128
|issue=20
|pages= 205106
|doi=10.1063/1.2908251
|pmid=18513049
|pmc=2809694}}
*{{cite journal
|author=Gregory R. Bowman, Xuhui Huang, Yuan Yao, Jian Sun, Gunnar Carlsson, Leonidas J. Guibas, and Vijay S. Pande
|title=Structural Insight into RNA Hairpin Folding Intermediates
|journal=Journal of the American Chemical Society
|year=2008
|volume=130
|issue=30
|pages=9676–9678
|doi=10.1021/ja8032857
|pmid=18593120
|pmc=2652247}}
*{{cite journal
|author=Nicholas W. Kelley, V. Vishal, Grant A. Krafft, and Vijay S. Pande.
|title=Simulating oligomerization at experimental concentrations and long timescales: A Markov state model approach.
|journal=Journal of Chemical Physics
|year=2008
|volume=129
|issue=21
|pages= 214707
|doi=10.1063/1.3010881
|pmid=19063575
|pmc=2674793}}
*{{cite journal
|author=Paula M. Petrone, Christopher D. Snow, Del Lucent, and Vijay S. Pande
|title=Side-chain recognition and gating in the ribosome exit tunnel
|journal=Proceedings of the National Academy of Sciences, USA
|year=2008
|volume=105
|issue=43
|pages=16549–16554
|doi=10.1073/pnas.0801795105
|pmid=18946046
|pmc=2575457}}
*{{cite journal
|author=Edgar Luttmann, Daniel L. Ensign, Vishal Vaidyanathan, Mike Houston, Noam Rimon, Jeppe Øland, Guha Jayachandran, Mark Friedrichs, Vijay S. Pande
|title=Accelerating Molecular Dynamic Simulation on the Cell processor and PlayStation 3
|journal=Journal of Computational Chemistry
|year=2008
|volume=30
|issue=2
|pages=268–274
|doi=10.1002/jcc.21054
|pmid=18615421}}

===2009===
*{{cite journal
|author=Peter M. Kasson and Vijay S. Pande
|title=Combining Mutual Information with Structural Analysis to Screen for Functionally Important Residues in Influenza Hemagglutinin
|journal=Pacific Symposium on Biocomputing
|year=2009
|volume=14
|issue=
|pages=492–503
|doi=
|pmid=19209725
|pmc=2811693}}
*{{cite journal
|author=Nicholas W. Kelley, Xuhui Huang, Stephen Tam, Christoph Spiess, Judith Frydman and Vijay S. Pande
|title=The predicted structure of the headpiece of the Huntingtin protein and its implications on Huntingtin aggregation
|journal=Journal of Molecular Biology
|year=2009
|volume=388
|issue=5
|pages=919–27
|doi=10.1016/j.jmb.2009.01.032
|pmid=19361448
|pmc=2677131}}
*{{cite journal
|author=M. S. Friedrichs, P. Eastman, V. Vaidyanathan, M. Houston, S. LeGrand, A. L. Beberg, D. L. Ensign, C. M. Bruns, V. S. Pande
|title=Accelerating molecular dynamic simulation on graphics processing units
|journal=Journal of Computational Chemistry
|year=2009
|volume=30
|issue=6
|pages=864–72
|doi=10.1002/jcc.21209
|pmid=19191337
|pmc=2724265}}
*{{cite journal
|author=D. L. Ensign and V. S. Pande
|title=The Fip35 WW Domain Folds with Structural and Mechanistic Heterogeneity in Molecular Dynamics Simulations
|journal=Biophysical Journal
|year=2009
|volume=96
|issue=8
|pages=L53–55
|doi=10.1016/j.bpj.2009.01.024
|pmid=19383445
|pmc=2718323}}
*{{cite journal
|author=V. A. Voelz, E. Luttmann, G. R. Bowman, and V.S. Pande
|title=Probing the nanosecond dynamics of a designed three-stranded beta-sheet with massively parallel molecular dynamics simulation
|journal=International Journal of Molecular Sciences
|year=2009
|volume=10
|issue=3
|pages=1013
|doi=10.3390/ijms10031013
|pmid=19399235
|pmc=2672016}}
*{{cite journal
|author=A. Beberg and V. S. Pande
|title=Folding@home: lessons from eight years of distributed computing
|journal=IEEE International Parallel and Distributed Processing Symposium
|year=2009
|volume=
|issue=
|pages=1–8
|doi=10.1109/IPDPS.2009.5160922
|pmid=}}
*{{cite journal
|author=G. R. Bowman, X. Huang, and V. S. Pande
|title=Using generalized ensemble simulations and Markov state models to identify conformational states
|journal=Methods
|year=2009
|volume=49
|issue=2
|pages=197–201
|doi=10.1016/j.ymeth.2009.04.013
|pmid=19410002
|pmc=2753735}}
*{{cite journal
|author=G. R. Bowman and V. S. Pande
|title=The Roles of Entropy and Kinetics in Structure Prediction
|journal=PLoS One
|year=2009
|volume=4
|issue=6
|pages=e5840
|doi=10.1371/journal.pone.0005840
|pmid=19513117
|pmc=2688754}}
*{{cite journal
|author=Peter M. Kasson, Daniel L. Ensign and Vijay S. Pande
|title=Combining Molecular Dynamics with Bayesian Analysis To Predict and Evaluate Ligand-Binding Mutations in Influenza Hemagglutinin
|journal=Journal of the American Chemical Society
|year=2009
|volume=131
|issue=32
|pages=11338–40
|doi=10.1021/ja904557w
|pmid=19637916
|pmc=2737089}}
*{{cite journal
|author=S. Bacallado, J. Chodera, and V. Pande
|title=Bayesian comparison of Markov models of molecular dynamics with detailed balance constraint
|journal=Journal of Chemical Physics
|year=2009
|volume=131
|issue=4
|pages=045106
|doi=10.1063/1.3192309
|pmid=19655927
|pmc=2730706}}

===2010===

*{{cite journal
|author=Vincent A. Voelz, Gregory R. Bowman, Kyle Beauchamp and Vijay S. Pande
|title=Molecular Simulation of ab Initio Protein Folding for a Millisecond Folder NTL9(1−39)
|journal=Journal of the American Chemical Society
|year=2010
|volume=132
|issue=5
|pages=1526–1528
|doi=10.1021/ja9090353
|pmid=20070076
|pmc=2835335}}
*{{cite journal
|author=P. M. Kasson, E. Lindahl, and V. S. Pande
|title=Atomic-Resolution Simulations Predict a Transition State for Vesicle Fusion Defined by Contact of a Few Lipid Tails
|journal=PLoS Computational Biology
|year=2010
|volume=6
|issue=6
|pages=e1000829
|doi=10.1371/journal.pcbi.1000829
|pmc=2891707
|pmid=20585620}}
*{{cite journal
|author=G. R. Bowman and V. S. Pande
|title=Protein folded states are kinetic hubs
|journal=Proceedings of the National Academy of Sciences, USA
|year=2010
|volume=107
|issue=24
|pages=10890–10895
|doi=10.1073/pnas.1003962107 
|pmc=
2890711|pmid=20534497}}
*{{cite journal
|author=V. S. Pande
|title=A simple theory of protein folding kinetics
|journal=Physical Review Letters
|year=2010
|volume=
|issue=
|pages=
|doi=
|pmc=
|pmid=}}
*{{cite journal
|author=EJ Thompson, AJ DePaul, SS Patel & EJ Sorin
|title=Evaluating Molecular Mechanical Potentials for Helical Peptides and Proteins.
|journal=PLoS ONE
|year=2010
|volume=5
|issue=
4|pages=e10056
|doi=10.1371/journal.pone.0010056.g001
|pmc=
2850926|pmid=20418937}}
*{{cite journal
|author=AJ DePaul, EJ Thompson, SS Patel, K Haldeman & EJ Sorin
|title=Equilibrium conformational dynamics in an RNA tetraloop from massively parallel molecular dynamics
|journal=Nucleic Acids Research
|year=2010
|volume=38
|issue=
14|pages=4856–4867
|doi=10.1093/nar/gkq134
|pmc=
2919701|pmid=20223768}}

==High performance platforms==
<!--[[Image:F@H FLOPS per client.svg|thumb|right|350px|Folding@home average megaFLOPS per individual client type. One single [[Graphics processing unit|GPU]] accounts for as many FLOPS as about 60 CPUs under Windows. Note that this is ''not'' a measure of raw processing power, which depends on a variety of other factors such as [[uptime]] and CPU load. Also, a client's FLOPS performance is only a measure of its speed, not necessarily its scientific usefulness.]]-->

===Graphical processing units===
On October 2, 2006, the Folding@home Windows GPU client was released to the public as a [[Development stage|beta test]]. After 9 days of processing from the Beta client the Folding@home project had received 31 tera[[FLOPS|FLOPs]] of computational performance from just 450 [[Radeon R520|ATI Radeon X1900]] GPUs, averaging at over 70x the performance of current CPU submissions, and the GPU clients remain the most powerful clients available in performance per client (as of March 11, 2009, GPU clients accounted for over 60% of the entire project's throughput at an approximate ratio of 9 clients per teraFLOP).<ref name="osstats"/>
On April 10, 2008, the second generation Windows GPU client was released to open beta testing, supporting ATI/AMD's Radeon HD 2000 and HD 3000 series, and also debuting a new core (GROGPU2 - Core 11). Inaccuracies with DirectX were cited as the main reason for the migration to the new version (the original GPU client was officially retired June 6, 2008<ref>{{cite web
|url = http://folding.typepad.com/news/2008/06/gpu1-has-been-retired-gpu2-for-nvidia-release-nearing.html
| title=Folding@home: GPU1 has been retired, GPU2 for NVIDIA release nearing}}</ref>), which uses AMD/ATI's [[Compute Abstraction Layer|CAL]]. On June 17, 2008, a version of the second-generation Windows GPU client for [[CUDA]] enabled Nvidia GPUs was also released for public beta testing.<ref>{{cite web
|url = http://folding.typepad.com/news/2008/06/gpu2-beta-client-for-nvidia-now-released.html
|title = Folding@home: GPU2 beta client for NVIDIA now released}}</ref> The GPU clients proved reliable enough to be promoted out of the beta phase and were officially released August 1, 2008.<ref>{{cite web
|url = http://folding.typepad.com/news/2008/08/new-clients-are-out-620.html
|title = Folding@home: New clients are out (6.20)|name=release620}}</ref> Newer GPU cores continue to be released for both CAL and CUDA.

While the only officially released GPU v2.0 client is for Windows, this client can be run on Linux under [[WINE (software)|Wine]] with NVIDIA graphics cards.<ref name="WINE wiki">{{cite web | url=http://gpu2.twomurs.com/index.php?title=Main_Page | title=Folding@Home GPU v2.0 Windows Client on Linux Wiki |date=2008-08-23 | accessdate=2008-11-06}} {{Dead link|date=September 2010|bot=H3llBot}}</ref> The client can operate on both 32- and 64-bit Linux platforms, but in either case the 32-bit CUDA toolkit is required. This configuration is not officially supported, though initial results have shown comparable performance to that of the native client and no problems with the scientific results have been found {{Citation needed|date=November 2008}}. An unofficial installation guide has been published.<ref name="WINE wiki"/>

On September 25, 2009, Vijay Pande revealed in his blog that a new third version of the GPU client was in development.<ref>{{cite web
| url = http://folding.typepad.com/news/2009/09/update-on-new-fah-cores-and-clients.html
| title = Folding@home: Update on new FAH core and clients}}</ref> GPU3 will use [[OpenCL]] (preferred over [[DirectCompute]]) as the software interface, which may mean that the GPU core will be unified for both ATI and nVidia, and may also mean the addition of support for other platforms with OpenCL support.

On May 25, 2010, Vijay Pande announced an open beta of the GPU3 client on the Folding@home blog:
<ref>{{cite web
| url = http://folding.typepad.com/news/2010/05/open-beta-release-of-the-gpu3-clientcore.html
| title = Folding@home: Open beta release of the GPU3 client/core}}</ref>
The new core initially only supports Nvidia GPUs, but will support ATI/AMD GPUs in a subsequent release.

===PlayStation 3===
{{main|Life with PlayStation}}
[[Image:LifeWithPlayStation Folding.jpg|right|thumb|220px|The PlayStation 3's ''Life With PlayStation'' client replaced the Folding@home application on September 18, 2008.]]
Folding@Home is also a channel of the application [[Life with PlayStation]] for [[PlayStation 3]].  The client was originally a standalone application, but is now part of a [[virtual globe]] which depicts [[news]], [[weather]] and [[encyclopedia|encyclopedic]] information (notably from [[Wikipedia]]).<ref name="osstats"/><ref>{{cite web| url= http://folding.stanford.edu/FAQ-PS3.html| title = PS3 FAQ| accessdate = 2006-11-13| author = Vijay Pande| date= 2006-10-22| publisher  = Stanford University}}</ref>

===Multi-core processing client===
<!--  Commented out: [[Image:FAH-SMP.jpg|thumb|150px|Folding@home SMP Client set to use 95% of a [[Multi-core (computing)|quad core]] processor.|{{ifdc|1=FAH-SMP.jpg|log=2009 July 15}}]] -->
As more modern CPUs are being released, the migration to [[Multi-core (computing)|multiple cores]] is becoming more adopted by the public, and the Pande Group is adding [[symmetric multiprocessing]] (SMP) support to the Folding@home client in the hopes of capturing the additional processing power. The SMP support is being achieved by utilizing [[Message Passing Interface]] (MPI) protocols. In current state it is being confined inside a single node by hard coded usage of the [[localhost]].

On November 13, 2006, the beta SMP Folding@home clients for [[x86-64]] [[Linux]] and x86 [[Mac OS X]] were released. The beta win32 SMP Folding@home client is out as well, and a 32-bit Linux client is currently in development.<ref>{{cite web
| url = http://folding.stanford.edu/FAQ-SMP.html
| title = Folding@home SMP Client FAQ
| accessdate = 2006-11-13
| author = Vijay Pande
| date = 2006-11-13
| publisher = Stanford University
}}</ref>

On June 17, 2009 the Pande Group revealed that a second generation SMP client (known as the SMP2 client) was in development. This client will use threads rather than MPI<ref name="fahdev"/> to spread the processing load across multiple cores and thereby remove the overhead of keeping the cores synced, as they should share a common data bank in RAM. On January 24, 2010, the first open beta release of the SMP2 client was made, trialling the new processing methods and a new points bonus system rewarding quick unit returns.

== Folding@home teams ==
A typical Folding@home user, running the client on a single PC, will likely not be ranked high on the list of contributors. However, if the user were to join a team, they would add the points they receive to a larger collective. Teams work by using the combined score of all their members. Thus, teams are ranked much higher than individual submitters. Rivalries between teams create friendly competition that benefits the folding community. Many teams publish their own stats, so members can have intra-team competitions for top spots.<ref>[http://forum.folding-community.org/viewtopic.php?t=8846 Folding-community: why have teams?]</ref> Some teams offer prizes in an attempt to increase participation in the project.<ref>{{cite web
| url = http://www.mprize.org/index.php?ctype=news&pagename=blogdetaildisplay&BID=2008032-20053630&detaildisplay=Y
| title = The Mprize-
}}</ref><ref>[http://jiggmin.com/forum/showthread.php?p=990473 Team Jiggmin F@H Prizes]</ref>

== Development ==

The Folding@home project does not make the project [[source code]] available to the public, citing security and integrity concerns.<ref>{{ cite web | url = http://folding.stanford.edu/English/FAQ-OpenSource | title = Folding@home Open Source FAQ }}</ref>  At the same time, the majority of the scientific codes used by the FAH (ex. [[Cosm]], [[GROMACS]], [[TINKER]], [[AMBER]], [[CPMD]], [[BrookGPU]]) are largely [[Open-source software]] or under similar licenses.

A development version of Folding@home once ran on the open source [[BOINC]] framework; however, this version remained unreleased.<ref>{{ cite web | url = http://folding.stanford.edu/English/FAQ-highperformance | title = FAH on BOINC | work = Folding@home high performance client FAQ }}</ref>

== Estimated energy consumption ==
Original PlayStation 3 has a maximum power rating of 380 [[watt]]s (newer versions have a lower rating). As Folding@home is a CPU intensive application, it causes 100% utilization. However, according to Stanford's PS3 FAQ, "We expect the PS3 to use about 200W while running Folding@home."<ref>{{cite web
| url = http://www.stanford.edu/group/pandegroup/folding/FAQ-PS3.html 
| title = PS3 FAQ
| format = FAQ
}}</ref>  
As of December 27, 2008, there are 55,291 PS3s providing 1,559,000,000 MFlops of processing power.  This amounts to 28,196 MFlops/PS3, and with Stanford's estimate of 200W per PS3 (for original units manufactured on the 90 nm process), 140.98 MFlops/watt.<ref name="osstats"/> This would put the PS3 portion of Folding@home at 95th on the November 2008 Green500 list.<ref>{{cite web
| url = http://green500.org/lists/listdisplay.php?month=11&year=2008&list=green500_200811.csv&start=1&line=101
| title =  Lists :: November 2008 :: Ranks 1-100
| work = Green 500
}}</ref>
The Cell processors used in 65 nm PlayStation 3s lower power consumption to around 140W per PS3, whilst the 45 nm PS3s reduce this again to around 100W.  This further increases the power efficiency of the contribution from PlayStation 3 units.  

The total power consumption required to produce the processing power required by the project can be estimated based upon the average [[FLOPS per watt]]. As of November 2008, according to the [[Green500]] list, the most efficient computer - also based on a version of the Cell BE - runs at 536.24 MFLOPS/watt.<ref>{{cite web
| url = http://green500.org/lists/2008/11/list.php
| title = The Green500 List
| accessdate = 2008-12-27
}}</ref> One petaFLOPS equals 1,000,000,000 MFLOPSs. Therefore, the current Folding@home project, if it were theoretically using the most efficient [[CPU]]s currently available, would use at least 2.8 [[megawatt]]s of power per petaFLOPS, slightly more than the world's first petaFLOPS system, the Cell-based Roadrunner which uses 2.345MW. This is equivalent to the power needed to light approximately 40,000 standard house light bulbs (between 60 and 100 watts each), or the equivalent of 1-3 [[wind turbine]]s depending on their size.<ref>{{cite web
| url = http://en.wikipedia.org/wiki/Windmill#Modern_windmills
| title = Windmill - Wikipedia, the free encyclopedia
}}</ref>

Estimates of energy usage per time period are more difficult than estimates of energy usage per processing instruction. This is because Folding@home clients are often run on computers that would be powered-on even in the absence of the Folding@home client, and that run other programs simultaneously. 

Looking at energy-balance in a larger context requires a number of assumptions. While Folding@home increases processor utilization, and thus (usually) power consumption; the extent to which it does so depends on the client processor's normal operating load, and its ability to reduce clock speeds when presented with less-than-full utilization (a process known as [[dynamic frequency scaling]]). In a Folding@home client that runs in a heated home, the excess heat generated by the power usage would reduce the amount of energy needed to heat the building to some degree. However, such energy gains would be offset over the year in those locations where air-conditioners are employed during the warmer months.

== See also ==
* [[Blue Gene]]
* [[Grid computing]]
* [[List of distributed computing projects]]
* [[List of software for molecular mechanics modeling|Software for molecular modeling]]
* [[Molecular modeling on GPU]]
* [[Rosetta@home]]

==References==
{{reflist|2}}

==External links==
*[http://folding.stanford.edu/ Folding@home project homepage]
*[http://fahwiki.net/index.php/Main_Page Folding@home Wiki]
*[http://www.fah-addict.net/ FAH-Addict; Folding@home related news]
*[http://kakaostats.com/ Kakao Folding@home Stats]
*[http://fah-web.stanford.edu/cgi-bin/main.py?qtype=teampage&teamnum=42223 Wikipedia team]
*[http://sites.google.com/site/techtdsn/folding-home 002 Tools F@H Performance Data] site disabled
*[http://www.parc.com/event/696/foldinghome.html Talk given by Folding@home author Vijay Pande at the PARC forum]
*[http://www.youtube.com/watch?v=2BVNCQt6MJw Folding@home Instructional Video on YouTube]
*[http://www.ustream.tv/recorded/1070617 Interview of Vijay Pande about Folding@Home Project]

{{DEFAULTSORT:Folding@Home}}
[[Category:Protein folds]]
[[Category:Protein structure]]
[[Category:Distributed computing projects]]
[[Category:Windows software]]
[[Category:Linux science software]]
[[Category:Mac OS X software]]
[[Category:PlayStation 3 software]]
[[Category:Cross-platform software]]

[[ar:فولدينغ@هوم]]
[[be:Folding@home]]
[[be-x-old:Folding@home]]
[[ca:Folding@home]]
[[da:Folding@home]]
[[de:Folding@home]]
[[es:Folding@home]]
[[eo:Folding@Home]]
[[fr:Folding@home]]
[[it:Folding@Home]]
[[he:Folding@home]]
[[lt:Folding@home]]
[[nl:Folding@home]]
[[ja:Folding@Home]]
[[no:Folding@Home]]
[[pl:Folding@home]]
[[pt:Folding@home]]
[[ru:Folding@home]]
[[fi:Folding@home]]
[[sv:Folding@home]]
[[tr:Folding@Home]]
[[uk:Folding@home]]
[[vi:Folding@home]]
[[zh-yue:Folding@home]]
[[zh:Folding@home]]</body> </html>