<html> <head> <title>Phosida</title></head><body>{{Orphan|date=February 2009}}
{{Unreferenced|date=May 2008}}
The [[Phosphorylation|'''PHO'''sphorylation]] '''SI'''te '''DA'''tabase '''PHOSIDA''' integrates thousands of high-confidence ''[[in vivo]]'' [[phosphosite]]s identified in various species on the basis of [[mass spectrometry]] technology. For each phosphosite, PHOSIDA lists matching [[kinase]] motifs, predicted secondary structures, conservation patterns, and its dynamic regulation upon stimulus or other treatments such as kinase inhibition, for example. It includes [[phosphoproteome]]s of various organisms ranging from [[eukaryote]]s such as human and [[yeast]] to [[bacteria]] such as ''[[Escherichia coli]]'' and ''[[Lactococcus lactis]]''. Even the [[phosphoproteome]] of an archaean organism, namely ''[[Halobacterium salinarium]]'', is available. The integration of phosphoproteomes identified in organisms, which cover the phylogenetic tree representatively, enables to examine phosphorylation events from a global point of view including conservation and evolutionary preservation in time.

Moreover, PHOSIDA also predicts phosphosites on the basis of [[support vector machine]]s.

==External links==
*[http://www.phosida.com Official website]

[[Category:Phosphorus]]
[[Category:Chemical databases]]</body> </html>