<html> <head> <title>Protomap (proteomics)</title></head><body>{{about|[[proteomics]]|the Protomap hypothesis (neuroscience)|Protomap (neuroscience)}}
'''PROTOMAP''' is a recently developed [[proteomic]] technology for identifying changes to proteins that manifest in altered migration by one-dimensional [[SDS-PAGE]].  It is similar, conceptually, to [[two-dimensional gel electrophoresis]] and [[difference gel electrophoresis]] in that it enables global identification of proteins that undergo altered [[electrophoresis|electrophoretic]] migration resulting from, for example, [[proteolysis]] or [[post-translational modification]].  However, it is unique in that all proteins are sequenced using [[mass spectrometry]] which provides information on the sequence coverage detected in each isoform of each protein thereby facilitating interpretation of proteolytic events. <ref name="pmid18775303">{{cite journal |author=Johnson CE, Kornbluth S |title=Caspase cleavage is not for everyone |journal=Cell |volume=134 |issue=5 |pages=720–1 |year=2008 |month=September |pmid=18775303 |doi=10.1016/j.cell.2008.08.019 |url=http://openurl.ebscohost.com/linksvc/linking.aspx?genre=article&sid=PubMed&issn=0092-8674&title=Cell&volume=134&issue=5&spage=720&atitle=Caspase%20cleavage%20is%20not%20for%20everyone.&aulast=Johnson&date=2008}}</ref> 

PROTOMAP is performed by resolving control and experimental samples in separate lanes of a 1D [[SDS-PAGE]] gel.  Each lane is cut into evenly spaced bands (usually 15-30 bands) and proteins in these bands are sequenced using [[shotgun proteomics]].  Sequence information from all of these bands are [[bioinformatics|bioinformatically]] integrated into a visual format called a '''peptograph''' which plots gel-migration in the vertical dimension (high- to low-molecular weight, top to bottom) and sequence coverage in the horizontal dimension (N- to C-terminus, left to right).  A peptograph is generated for each protein the sample (thousands of peptographs are generated from a single experiment) and this data format enables rapid identification of proteins undergoing proteolytic cleavage by making evident changes in gel-migration that are accompanied by altered topography.

PROTOMAP stands for '''PRotein TOpography and Migration Analysis Platform''' and was invented and developed by [[Ben Cravatt]] and colleagues at [[The Scripps Research Institute]].<ref name="pmid18724940">{{cite journal |author=Dix MM, Simon GM, Cravatt BF |title=Global mapping of the topography and magnitude of proteolytic events in apoptosis |journal=Cell |volume=134 |issue=4 |pages=679–91 |year=2008 |month=August |pmid=18724940 |doi=10.1016/j.cell.2008.06.038 |url=http://www.cell.com/retrieve/pii/S0092867408008258 |pmc=2597167}}</ref>

==See also==
* [[Two dimensional SDS-PAGE]]
* [[Difference gel electrophoresis]]
* [[Proteomics]]
* [[Shotgun proteomics]]
* [[Mass spectrometry]]
* [[Protease]]
* [[Benjamin Cravatt III|Benjamin Cravatt]]

==External links==
* [http://www.scripps.edu/chemphys/cravatt/protomap The PROTOMAP website] - provides access to publicly available PROTOMAP data as well as software and protocols for running PROTOMAP experiments.
* [http://www.scripps.edu/chemphys/cravatt/research.html#protomap PROTOMAP Description] - on the Cravatt Lab website

==References==
{{Reflist|2}}

[[Category:Laboratory techniques]]
[[Category:Electrophoresis]]
[[Category:Proteomics]]
[[Category:Bioinformatics]]
[[Category:Bioinformatics software]]
[[Category:Proteins]]
[[Category:Mass spectrometry]]
[[Category:Mass spectrometry software]]
[[Category:The Scripps Research Institute]]</body> </html>