<html> <head> <title>Tree rearrangement</title></head><body>'''Tree rearrangements''' are used in [[heuristic]] [[algorithm]]s devoted to searching for an [[Optimization (mathematics)|optimal]] [[tree structure]]. They can be applied to any set of data that are naturally arranged into a tree, but have most applications in [[computational phylogenetics]], especially in [[maximum parsimony]] and [[maximum likelihood]] searches of [[phylogenetic tree]]s, which seek to identify one among many possible trees that best explains the [[evolution]]ary history of a particular [[gene]] or [[species]].



==Basic tree rearrangements==
<gallery>
Image:NNI.svg|Nearest Neighbor Interchange (NNI)
Image:SPR.svg|Sub-tree Pruning and Regrafting (SPR)
Image:TBR.svg|Tree bisection and reconnection (TBR)
</gallery>

The simplest tree-rearrangement, known as ''nearest-neighbor interchange'', exchanges the connectivity of four subtrees within the main tree. Because there are three possible ways of connecting four subtrees,<ref name="felsenstein">Felsenstein J. (2004). ''Inferring Phylogenies'' Sinauer Associates: Sunderland, MA.</ref> and one is the original connectivity, each interchange creates two new trees. Exhaustively searching the possible nearest-neighbors for each possible set of subtrees is the slowest but most optimizing way of performing this search. An alternative, more wide-ranging search, ''subtree pruning and regrafting'' (SPR), selects and removes a subtree from the main tree and reinserts it elsewhere on the main tree to create a new node. Finally, ''tree bisection and reconnection'' (TBR) detaches a subtree from the main tree at an interior node and then attempts all possible connections between branches of the two trees thus created. The increasing complexity of the tree rearrangement technique correlates with increasing computational time required for the search, although not necessarily with their performance.<ref name="takahashi">Takahashi K, Nei M. (2000). Efficiencies of fast algorithms of phylogenetic inference under the criteria of maximum parsimony, minimum evolution, and maximum likelihood when a large number of sequences are used. ''Mol Biol Evol'' 17(8):1251-8.</ref>

==Tree fusion==
The simplest type of tree fusion begins with two trees already identified as near-optimal; thus, they most likely have the majority of their nodes correct but may fail to resolve individual tree "leaves" properly; for example, the separation ((A,B),(C,D)) at a branch tip versus ((A,C),(B,D)) may be unresolved.<ref name="felsenstein" /> Tree fusion swaps these two solutions between two otherwise near-optimal trees. Variants of the method use standard [[genetic algorithm]]s with a defined [[objective function]] to swap high-scoring subtrees into main trees that are high-scoring overall.<ref name="matsuda">Matsuda H. (1996). Protein phylogenetic inference using maximum likelihood with a genetic algorithm. ''Pacific Symposium on Biocomputing 1996'', pp512-23.</ref>

==References==
<references />

[[Category:Phylogenetics]]
[[Category:Optimization algorithms]]
[[Category:Trees (structure)]]</body> </html>