<html> <head> <title>Treefinder</title></head><body>'''Treefinder''' is a free computer program for the [[maximum likelihood|likelihood-based]] reconstruction of [[phylogenetic tree]]s from molecular sequences. A platform-independent graphical environment integrates a standard suite of analyses that can be done by just pressing a few buttons: [[computational phylogenetics|phylogeny reconstruction]], bootstrap analysis, [[model selection]], [[hypothesis testing]], tree calibration, manipulation of trees and sequence data. Treefinder is also programmable in its own [[scripting language]] TL.

Treefinder has an efficient tree search [[algorithm]] that can infer even large trees with thousands of [[species]] within a short time. Result trees are displayed on the screen and can then be saved in a special data format, the "reconstruction report", which may serve as an input for further analysis, for example hypothesis testing. The report contains all information about the tree and the models used. Export of results to popular tree formats such as [[newick format|NEWICK]] or [[nexus format|NEXUS]] is also possible.

Noteworthy is the broad collection of [[substitution model|models of sequence evolution]]. The version of June 2008 implements 7 models of nucleotide substitution (HKY, TN, J1, J2, J3 (= TIM), TVM, GTR), 14 empirical models of amino acid substitution (BLOSUM, cpREV, Dayhoff, JTT, LG, mtArt, mtMam, mtREV, PMB, rtREV, betHIV, witHIV, VT, WAG), 4 substitution models of structured rRNA (bactRNA, eukRNA, euk23RNA, mitoRNA), the 6-state "Dayhoff Groups" protein model (DG), 2-state and 3-state models of DNA (GTR3, GTR2), a parametric mixed model (MIX) mixing the empirical models of proteins or rRNA, and also a user-definable GTR-type model (MAP) mapping characters to states as needed. Three models of among-site rate heterogeneity are available (Gamma,Gamma+I,I), which can be combined with any of the substitution models. Last but not least, one can assume different models for different partitions of a [[sequence alignment]], and partitions may be assumed to evolve at different speeds. All parameters of the models can be estimated from the data by maximization of likelihood. Special TL expressions, the "model expressions", allow the concise notation of even complex models, together with all their parameters and optimization modes.

Treefinder <ref>Jobb G, von Haeseler A, Strimmer K. 2004. TREEFINDER: a powerful graphical analysis environment for molecular phylogenetics. BMC Evol. Biol. 4:18.</ref> has been produced by Gangolf Jobb, formerly a researcher at the University of Munich, Germany.

==References==
<references/> 

==See also==
* [[Computational phylogenetics]]
* [[Molecular phylogeny]]
* [[Molecular biology]]

==External links==
* [http://www.treefinder.de/ Treefinder homepage]

[[Category:Computational phylogenetics]]
[[Category:Bioinformatics software]]
[[Category:Science software]]
[[Category:Molecular biology]]
[[Category:Genetics]]</body> </html>