<html> <head> <title>Biochip</title></head><body>{{Context|date=October 2009}}

[[Image:Biochip.jpg|thumb|Hundreds of gel drops are visible on the biochip]]
The development of '''biochips''' is a major thrust of the rapidly growing [[biotechnology]] industry, which encompasses a very diverse range of
research efforts including [[genomics]], [[proteomics]], and [[pharmaceuticals]], among other activities. Advances in
these areas are giving scientists new methods for unravelling the complex
biochemical processes occurring inside cells, with the larger goal of
understanding and treating human diseases. At the same time, the
[[semiconductor industry]] has been steadily perfecting the science of
micro-miniaturization. The merging of these two fields in recent years has
enabled biotechnologists to begin packing their traditionally bulky
sensing tools into smaller and smaller spaces, onto so-called biochips. These
chips are essentially miniaturized laboratories that can perform hundreds or
thousands of simultaneous biochemical reactions. Biochips enable researchers
to quickly screen large numbers of biological analytes for a variety of
purposes, from disease diagnosis to detection of bioterrorism agents.

==History==
The development of biochips has a long history, starting with early work on
the underlying [[sensor]] technology. One of the first portable, chemistry-based
sensors was the [[glass electrode|glass pH electrode]], invented in 1922 by
Hughes (Hughes, 1922). Measurement of [[pH]] was accomplished by
detecting the potential difference developed across a thin glass membrane
selective to the permeation of hydrogen ions; this selectivity was achieved
by exchanges between H<sup>+</sup> and SiO sites in the glass. The basic concept of
using exchange sites to create permselective membranes was used to develop
other [[Ion selective electrode|ion sensors]] in subsequent years. For example, a K<sup>+</sup> sensor was
produced by incorporating [[valinomycin]] into a thin membrane (Schultz, 1996).
Over thirty years elapsed before the first true [[biosensor]] (''i.e.'' a
sensor utilizing biological molecules) emerged. In 1956, Leland Clark
published a paper on an [[Clark electrode|oxygen sensing electrode]] (Clark, 1956_41).
This device became the basis for a [[glucose]] sensor developed in 1962 by Clark
and colleague Lyons which utilized [[glucose oxidase]] molecules embedded in a
[[dialysis]] membrane (Clark, 1962). The [[enzyme]] functioned in the
presence of glucose to decrease the amount of oxygen available to the oxygen electrode, thereby relating oxygen levels to glucose concentration. This and
similar biosensors became known as enzyme electrodes, and are still in use
today.

In 1953, [[Watson and Crick]] announced their discovery of the now familiar
[[double helix]] structure of [[DNA]] molecules and set the stage for [[genetics]]
research that continues to the present day (Nelson, 2000). The development
of [[sequencing]] techniques in 1977 by [[Walter Gilbert|Gilbert]] (Maxam, 1977) and
[[Frederick Sanger|Sanger]] (Sanger, 1977) (working separately) enabled researchers to
directly read the genetic codes that provide instructions for
[[protein biosynthesis|protein synthesis]]. This research showed how [[hybridisation (molecular biology)|hybridization]] of complementary single
[[oligonucleotide]] strands could be used as a basis for DNA sensing. Two
additional developments enabled the technology used in modern DNA-based
biosensors. First, in 1983 [[Kary Mullis]] invented the
[[polymerase chain reaction]]
(PCR) technique (Nelson, 2000), a method for amplifying DNA concentrations.
This discovery made possible the detection of extremely small quantities of
DNA in samples. Second, in 1986 Hood and co-workers devised a method to label
DNA molecules with [[fluorescent tag]]s instead of
radiolabels (Smith, 1986), thus enabling hybridization experiments to
be observed optically.

The rapid technological advances of the [[biochemistry]] and [[semiconductor]] fields
in the 1980s led to the large scale development of biochips in the 1990s.
At this time, it became clear that biochips were largely a "platform"
technology which consisted of several separate, yet integrated components.
Figure 1 shows the make up of a typical biochip platform.
The actual sensing component (or "chip") is just one piece of a complete
analysis system. [[Transducer|Transduction]] must be done to translate the actual sensing
event (DNA binding, [[redox|oxidation/reduction]], ''etc.'') into a format
understandable by a computer ([[voltage]], light intensity, mass, ''etc.''),
which then enables additional analysis and processing to produce a final,
[[human-readable]] output. The multiple technologies needed to make a successful
biochip &mdash; from sensing chemistry, to [[microarray]]ing, to signal processing &mdash;
require a true multidisciplinary approach, making the barrier to entry steep.
One of the first commercial biochips was introduced by [[Affymetrix]]. Their
"GeneChip" products contain thousands of individual DNA sensors for use in
sensing defects, or single nucleotide polymorphisms (SNPs), in genes such as
[[p53]] (a tumor suppressor) and [[BRCA1]] and [[BRCA2]] (related to breast
cancer) (Cheng, 2001). The chips are produced using [[microlithography]]
techniques traditionally used to fabricate [[integrated circuits]] (see below).

[[image:Biochip platform.jpg|frame|thumb right|300 px|Figure 1. Biochips are a platform that require, in addition to microarray technology, transduction and signal processing
technologies to output the results of sensing experiments.]]

Today, a large variety of biochip technologies are either in development or
being commercialized. Numerous advancements continue to be made in sensing
research that enable new platforms to be developed for new applications.
Cancer diagnosis through DNA typing is just one market opportunity. A variety
of industries currently desire the ability to simultaneously screen for a
wide range of chemical and biological agents, with purposes ranging from
testing public water systems for disease agents to screening airline cargo
for explosives. Pharmaceutical companies wish to combinatorially screen drug
candidates against target enzymes. To achieve these ends, [[DNA]], [[RNA]], [[proteins]],
and even living cells are being employed as sensing mediators on biochips (Potera, 2008).
Numerous transduction methods can be employed including [[surface plasmon resonance]], [[fluorescence]], and [[chemiluminescence]]. The particular sensing and
transduction techniques chosen depend on factors such as price, sensitivity, and reusability.

==Microarray fabrication==
The microarray &mdash; the dense, two-dimensional grid of biosensors &mdash; is the critical component of a biochip platform. Typically, the sensors are deposited on a flat substrate, which may either be passive (''e.g.'' silicon or glass) or active, the latter
consisting of integrated electronics or [[microtechnology|micromechanical]] devices that perform or assist signal transduction. [[Surface chemistry]] is used to [[covalent bond|covalently bind]] the sensor molecules to the substrate medium. The fabrication of microarrays is non-trivial and is a major economic and technological hurdle that may
ultimately decide the success of future biochip platforms. The primary manufacturing challenge is the process of placing each sensor at a specific position (typically on a [[Cartesian coordinate system|Cartesian]] grid) on the substrate. Various means exist to achieve the placement, but typically robotic micro-pipetting (Schena, 1995) or micro-printing (MacBeath, 1999) systems are used to place tiny spots of sensor material on the chip surface. Because each sensor is unique, only a few spots can be placed at a time. The low-throughput nature of this
process results in high manufacturing costs.

Fodor and colleagues developed a unique fabrication process (later used by
[[Affymetrix]]) in which a series of microlithography steps is used to
[[Combinatorial chemistry|combinatorially synthesize]] hundreds of thousands of unique, single-stranded
DNA sensors on a substrate one [[nucleotide]] at a
time (Fodor, 1991; Pease, 1994). One lithography step is needed per base type; thus, a total
of four steps is required per nucleotide level. Although this technique is
very powerful in that many sensors can be created simultaneously, it is
currently only feasible for creating short DNA strands (15&ndash;25 nucleotides).
Reliability and cost factors limit the number of photolithography steps that
can be done. Furthermore, light-directed combinatorial synthesis techniques
are not currently possible for proteins or other sensing molecules.

As noted above, most microarrays consist of a Cartesian grid of sensors. This
approach is used chiefly to map or "encode" the coordinate of each sensor
to its function. Sensors in these arrays typically use a universal signalling
technique (''e.g.'' fluorescence), thus making coordinates their only
identifying feature. These arrays must be made using a serial process
(''i.e.'' requiring multiple, sequential steps) to ensure that each sensor
is placed at the correct position.

"Random" fabrication, in which the sensors are placed at arbitrary
positions on the chip, is an alternative to the serial method. The tedious and expensive positioning process is
not required, enabling the use of parallelized self-assembly techniques. In
this approach, large batches of identical sensors can be produced; sensors
from each batch are then combined and assembled into an array. A
non-coordinate based encoding scheme must be used to identify each sensor. As
the figure shows, such a design was first demonstrated (and later
commercialized by Illumina) using functionalized beads placed randomly in the
wells of an etched [[fiber optic]]
cable (Steemers, 2000; Michael, 1998) Each bead was uniquely
encoded with a fluorescent signature. However, this encoding scheme is
limited in the number of unique dye combinations that can be used and
successfully differentiated.

==Protein biochip array and other microarray technologies==
[[Microarray]]s are not limited to [[DNA]] analysis; [[protein microarray]]s, [[antibody microarray]], [[chemical compound microarray]] can also be produced using biochips.  [http://www.randox.com Randox] Laboratories Ltd. launched Evidence, the first protein Biochip Array Technology analyzer in 2003.  In protein Biochip Array Technology, the biochip replaces the [[ELISA]] plate or [[cuvette]] as the reaction platform.  The biochip is used to simultaneously analyze a panel of related tests in a single sample, producing a [[patient]] profile.  The patient profile can be used in disease screening, [[medical diagnosis|diagnosis]], monitoring disease progression or monitoring treatment.  Performing multiple analyses simultaneously, described as multiplexing, allows a significant reduction in processing time and the amount of patient sample required.  Biochip Array Technology is a novel application of a familiar methodology, using sandwich, competitive and antibody-capture [[immunoassay]]s.  The difference from conventional immunoassays is that the capture ligands are covalently attached to the surface of the biochip in an ordered array rather than in solution.

In sandwich assays an enzyme-labelled antibody is used; in competitive assays an enzyme-labelled antigen is used.  On antibody-antigen binding a [[chemiluminescence]] reaction produces light.  Detection is by a [[charge-coupled device]] (CCD) camera.  The CCD camera is a sensitive and high-resolution sensor able to accurately detect and quantify very low levels of light.  The test regions are located using a grid pattern then the chemiluminescence signals are analysed by imaging software to rapidly and simultaneously quantify the individual analytes.

Details about other array technologies can be found in the following page: [[Antibody microarray]]

==See also==
* [[DNA microarray]]
* [[Protein array]]
* [[Chemical compound microarray]]
* [[Antibody microarray]]
* [[Tissue microarray]]
* [[Single nucleotide polymorphism]]
* [[Sequencing]]
* [[Lab-on-a-chip]]
* [[Electrophysiology#Planar patch clamp|Planar Patch Clamp]]
* [[Nanosensors]]
* [[Magnetic immunoassay]]

==References==
<!-- This article does not use the ref-ref method.  Please add new references according to precedent set by previous editors. -->
* Vahid Bemanian, Frøydis D. Blystad, Live Bruseth, Gunn A. Hildrestrand, Lise Holden, Endre Kjærland, Pål Puntervoll, Hanne Ravneberg and Morten Ruud, "What is Bioethics?" Dec 1998.
* M. Burnham, R. Mitchell, " Bioethics &mdash; An Introduction" 1992.
* {{cite book |author= Cady, NC|year=2009|chapter=Microchip-based PCR Amplification Systems|title=Lab-on-a-Chip Technology: Biomolecular Separation and Analysis|publisher=Caister Academic Press|isbn= 978-1-904455-47-9}}
* L. C. Clark, Jr., “Monitor and control of blood tissue O2 tensions,” ''Transactions of the American Society for Artificial Internal Organs'' 2, pp. 41–84, 1956.
* L. C. Clark, Jr. and C. Lyons, “Electrode system for continuous monitoring in cardiovascular surgery,” ''Annals of the New York Academy of Sciences'' 148, pp. 133–153, 1962.
* {{cite book |author= Fan et al.|year=2009|chapter=Two-Dimensional Electrophoresis in a Chip|title=Lab-on-a-Chip Technology: Biomolecular Separation and Analysis|publisher=Caister Academic Press|isbn= 978-1-904455-47-9}}
* S. P. Fodor, J. L. Read, M. C. Pirrung, L. Stryer, A. T. Lu, and D. Solas, “Light-directed, spatially addressable parallel chemical analysis,” ''Science'' 251, pp. 767–773, 1991.
* P. Fortina, D. Graves, C. Stoeckert, Jr., S. McKenzie, and S. Surrey in ''Biochip Technology'', J. Cheng and L. J. Kricka, eds., ch. Technology Options and Applications of DNA Microarrays, pp. 185–216, Harwood Academic Publishers, Philadelphia, 2001.
* K. L. Gunderson, S. Kruglyak, M. S. Graige, F. Garcia, B. G. Kermani, C. Zhao, D. Che, T. Dickinson, E. Wickham, J. Bierle, D. Doucet, M. Milewski, R. Yang, C. Siegmund, J. Haas, L. Zhou, A. Oliphant, J.-B. Fan, S. Barnard, and M. S. Chee, “Decoding randomly ordered DNA arrays,” ''Genome Research'' 14(5), pp. 870–877, 2004.
*{{cite book |author= Herold, KE; Rasooly, A (editor)| year=2009 |title=Lab-on-a-Chip Technology: Fabrication and Microfluidics | publisher=Caister Academic Press | isbn= 978-1-904455-46-2}}
*{{cite book |author= Herold, KE; Rasooly, A (editor)| year=2009 |title=Lab-on-a-Chip Technology: Biomolecular Separation and Analysis | publisher=Caister Academic Press | isbn= 978-1-904455-47-9}}
* W. S. Hughes, “The potential difference between glass and electrolytes in contact with water,” ''J. Am. Chem. Soc.'' 44, pp. 2860–2866, 1922.
* A. M. Maxam and W. Gilbert, “A new method for sequencing DNA,” ''Proc. Nat. Acad. Sci.'' 74, pp. 560–564, 1977.
* G. MacBeath, A. N. Koehler, and S. L. Schreiber, “Printing small molecules as microarrays and detecting protein-ligand interactions en masse,” ''J. Am. Chem. Soc.'' 121, pp. 7967–7968, 1999.
* K. L. Michael, L. C. Taylor, S. L. Schultz, and D. R. Walt, “Randomly ordered addressable high-density optical sensor arrays,” ''Analytical Chemistry'' 70, pp. 1242–1248, 1998.
* D. L. Nelson and M. M. Cox, ''Lehninger Principles of Biochemistry'', Worth Publishers, New York, 2000.
* {{Cite news
 | last = Potera | first = Carol | publication-date = 1 September 2008
 | title = Delivery of Time-Lapsed Live-Cell Imaging
 | periodical = [[Genetic Engineering & Biotechnology News]] | series = GEN Biobusiness | issn = 1935-472X
 | volume = 28 | issue = 15 | pages = 14
 | url = http://www.genengnews.com/biobusiness/cpitem.aspx?aid=2576 | accessdate = 29 April 2009
}}
* A. C. Pease, D. Solas, E. J. Sullivan, M. T. Cronin, C. P. Holmes, and S. P. Fodor, “Light-generated oligonucleotide arrays for rapid DNA sequence analysis,” ''Proc. Natl. Acad. Sci.'' 91, pp. 5022–5026, 1994.
* C. Roberts, C. S. Chen, M. Mrksich, V. Martichonok, D. E. Ingber, and G. M. Whitesides, “Using mixed self-assembled monolayers presenting RGD and (EG)3OH groups to characterize long-term attachment of bovine capillary endothelial cells to surfaces,” ''J. Am. Chem. Soc.'' 120, pp. 6548–6555, 1998.
* F. Sanger, S. Nicklen, and A. R. Coulson, “DNA sequencing with chainterminating inhibitors,” ''Proc. Nat. Acad. Sci.'' 74, pp. 5463–5467, 1977.
* M. Schena, D. Shalon, R. W. Davis, and P. O. Brown, “Quantitative monitoring of gene expression patterns with a complementary DNA microarray,” ''Science'' 270, pp. 467–470, 1995.
* H. Schmeck, "Blazing the Genetic Trail." Bethesda, MD: Howard Hughes Medical Institute, 1991.
* J. S. Schultz and R. F. Taylor in ''Handbook of Chemical and Biological Sensors'', J. S. Schultz and R. F. Taylor, eds., ch. Introduction to Chemical and Biological Sensors, pp. 1–10, Institute of Physics Publishing, Philadelphia, 1996.
* L. M. Smith, J. Z. Sanders, R. J. Kaiser, P. Hughes, C. Dodd, C. R. Connell, C. Heiner, S. B. H. Kent, and L. E. Hood, “Fluorescence detection in automated DNA sequence analysis,” ''Nature'' 321, pp. 61–67, 1986.
* F. J. Steemers, J. A. Ferguson, and D. R. Walt, “Screening unlabeled DNA targets with randomly-ordered fiber-optic gene arrays,” ''Nature Biotechnology'' 18, pp. 91–94, 2000.
* [http://www.accessexcellence.org/RC/AB/IE/Future_Of_Genetic_Research.html The Future of Genetic Research]
* [http://web.reed.edu/reed_magazine/spring06/features/life_in_venice/should_we.html Interview of A. Caplan, "Should We or Shouldn't We?"]
* [http://www.woodrow.org/teachers/bi/1992/bioethics_intro.html Bioethics Intro]
* NBIAP NEWS REPORT, U.S. Department of Agriculture, "To Regulate or Not to Regulate"  Forum: To Rationalize U.S. Biotech Regs.  June 1994
* [http://www.accessexcellence.org/RC/AB/IE/To_Regulate_or_Not.html To Regulate or Not to Regulate]
* [http://searchcio-midmarket.techtarget.com/sDefinition/0,,sid183_gci211664,00.html What is a Biochip?]

[[Category:Bioinformatics]]
[[Category:Biotechnology]]
[[Category:Molecular biology]]
[[Category:Genomics]]
[[Category:Proteomics]]
[[Category:Microarrays]]

[[ar:رقاقة حيوية]]
[[ca:Bioxip]]
[[da:Bio-chip]]
[[de:Biochip]]
[[es:Biochip]]
[[mn:Биочип]]
[[pl:Biochip]]
[[uk:Біочіп]]
[[zh:生物晶片]]</body> </html>