<html> <head> <title>Xrate</title></head><body>{{Infobox Software|name=XRATE|developer=Ian Holmes ([[UC Berkeley]])|latest_release_version=1|operating_system=[[UNIX]], [[Linux]], [[Apple Macintosh|Mac]], [[Cygwin]] on [[Windows XP]]|genre=Bioinformatics tool|licence=Open source|website=[http://biowiki.org/XRATE XRate homepage]}}

'''XRATE''' is a program for prototyping [[phylogenetic]] [[hidden Markov models]] and [[stochastic context-free grammar|stochastic context-free grammars]].
It is used to discover patterns of evolutionary conservation in [[sequence alignment|sequence alignments]].
The program can be used to estimate parameters for such models from "training" alignment data,
or to apply the parameterized model so as to annotate new alignments.
The program allows specification of a variety of [[Models of DNA evolution|models of DNA sequence evolution]] which may be arbitrarily organized using [[Formal grammar|formal grammars]].

As an example of how '''XRATE''' is used, consider a protein-coding [[gene]] consisting of [[exons]] interspersed with [[introns]].
The exons contain triplets of nucleotides ([[codons]]) that are translated by [[ribosomes]] according to the [[genetic code]], and consequently are under [[selection pressure]]
(since any mutation may affect the translated [[amino acid sequence]]).
In contrast, the introns are under fewer selective constraints and tend to evolve faster.
These varying pressures show up clearly in [[multiple sequence alignment|multiple alignments]].
The sequential layout of introns and exons can be described using [[Formal grammar|grammar theory]] (from linguistics)
and each of their distinct evolutionary signatures modeled as a [[continuous-time Markov process]].
'''XRATE''' allows the user to specify such models in a configuration file and estimate their parameters (evolutionary rates, length distributions of exons and introns, etc.)
directly from alignment data, using the [[Expectation-maximization algorithm]].

'''XRATE''' can be downloaded as part of the [http://biowiki.org/DART DART software package].  It accepts input files in [[Stockholm format]].

== References ==

* Holmes I, Rubin GM.  [http://www.ncbi.nlm.nih.gov/pubmed/11955022?dopt=Abstract An expectation maximization algorithm for training hidden substitution models.]  J Mol Biol. 2002 Apr 12;317(5):753-64.
* Klosterman PS, Uzilov AV, Bendana YR, Bradley RK, Chao S, Kosiol C, Goldman N, Holmes I.  [http://www.biomedcentral.com/1471-2105/7/428 XRate: a fast prototyping, training and annotation tool for phylo-grammars].  BMC Bioinformatics. 2006 Oct 3;7:428.

== External links ==
* '''XRATE''' [http://biowiki.org/XRATE homepage] and [http://biowiki.org/XrateFormat file format]

[[Category:Bioinformatics software]]</body> </html>