<html> <head> <title>DNA microarray experiment</title></head><body>[[Image:Microarray.svg|thumb|300px|steps involved in a microarray experiment (some steps omitted)]]

{{For|DNA microarrays in general|DNA microarray}}
This is an example of a '''DNA microarray experiment''', detailing a particular case to better explain DNA microarray experiments, while enumerating possible alternatives.

----

# The two samples to be compared (pairwise comparison) are grown/acquired. In this example treated sample ([[Case-control|case]]) and untreated sample ([[Case-control|control]]).
# The [[nucleic acid]] of interest is purified: this can be all [[RNA]] for [[expression profiling]], [[DNA]] for [[comparative hybridization]], or DNA/RNA bound to a particular [[protein]] which is [[immunoprecipitated]] ([[ChIP-on-chip]]) for [[Epigenetics |epigenetic]] or regulation studies. In this example total RNA is isolated (total as it is nuclear and [[cytoplasm]]ic) by [[Guanidinium thiocyanate-phenol-chloroform extraction]] (e.g. [[Trizol]]) which isolates most RNA (whereas column methods have a cut off of 200 nucleotides) and if done correctly has a better purity.
# The purified RNA is analysed for quality (by [[capillary electrophoresis]]) and quantity (by using a [[spectrometer|nanodrop spectrometer]]): if enough material (>1[[μg]]) is present the experiment can continue.
# The labelled product is generated via [[reverse transcription]] and sometimes with an optional [[PCR]] amplification. The RNA is reverse transcribed with either polyT primers which amplify only [[MRNA|mRNA]] or random primers which amplify all RNA which is mostly [[RRNA|rRNA]], [[MicroRNA|miRNA]] microarray ligate an oligonucleotide to the purified small RNA (isolated with a fractionator) and then RT and amplified. The label is added either in the RT step or in an additional step after amplification if present.The sense that is labelled depends on the microarray, which means that if the label is added with the RT mix, the [[CDNA|cDNA]] is on the template strand while the probe is on the sense strand (unless they are negative controls). The label is typically [[fluorescent]]; only one machine uses [[radioactivity in biology|radiolabels]]. The labelling can be direct (not used) or indirect which requires a coupling stage. The coupling stage can occur before hybridization (two-channel arrays) using [[aminoallyl]]-UTP and [[N-hydroxysuccinimide|NHS]] amino-reactive dyes (like [[cyanine|cyanine dyes]]) or after (single-channel arrays) using biotin and labelled streptavin. The modified nucleotides (typically a 1 aaUTP: 4 TTP mix) are added enzymatically at a lower rate compared to normal nucleotides, typically resulting in 1 every 60 bases (measured with a spectrophotometer). The aaDNA is then purified with a [[DNA separation by silica adsorption|column]] (using solution containing phosphate buffer as Tris contains amine groups). The aminoallyl group is an amine group on a long linker attached to the nucleobase, which reacts with a reactive dye. A dye flip is a type of replicate done to remove any dye effects in two-channel dyes, in one slide one same is labeled with Cy3 the other with Cy5, this is reversed in a different slide. In this example, in the presence of [[aminoallyl]]-UTP added in the RT mix.
# The labeled samples are then mixed with a propriety [[nucleic acid hybridization|hybridization]]  solution which may contain [[Sodium dodecyl sulfate|SDS]], [[citrate|SSC]], [[dextran|dextran sulfate]], a blocking agent (such as COT1 DNA, salmon sperm DNA, calf thymus DNA, [[PolyA]] or PolyT), [[Denhardt's solution]] and [[Methylamine|formamine]].
# This mix is denatured and added to a pin hole in a microarray, which can be a [[gene chip]] (holes in the back) or a glass microarray which is bound by a cover, called a mixer, containing two pinholes and sealed with the slide at the perimeter.
# The holes are sealed and the microarray hybridized, either in a hyb oven, where the microarray is mixed by rotation, or in a mixer, where the microarray is mixed by alternating pressure at the pinholes.
# After an overnight hybridization, all nonspecific binding is washed off (SDS and SSC).
# The microarray is dried and scanned in a special machine where a laser excites the dye and a detector measures its emission.
# The image is gridded with a template and the intensities of the features (several pixels make a feature) are quantified.
# The raw data is normalized, the simplest way is to subtract the background intensity and then divide the intensities making either the total intensity of the features on each channel equal or the intensities of a reference gene and then the [[t-value]] for all the intensities is calculated. More sophisticated methods include [[z-score|z-ratio]], [[local regression|loess and lowess regression]] and RMA (robust multichip analysis) for Affymetrix chips (single-channel, silicon chip, in situ synthesised short oligonucleotides).

==See also==
*[[DNA microarray]]
*[[Microfluidics]] or [[lab-on-chip]]
*[[Quantification of nucleic acids]]
*[[Cyanine]] dyes, such as Cy3 and Cy5
*[[Protocol (natural sciences)]]
*[[Replication|sample replication]]
*[[Molecular biology]]
*[[Serial analysis of gene expression]]


== External links ==
* [http://learn.genetics.utah.edu/content/labs/microarray/ DNA microarray virtual experiment]

{{Molecular Biology}}

==References==
* Gibson and Muse, A primer of genome science etc ISBN 0-87893-232-1
* Chomczynski, P. & Sacchi, N. Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction:Twenty-something years on. [http://dx.doi.org/10.1038/nprot.2006.83 Nature Prot. 1, 581–585 (2006)].
* Sambrook and Russell (2001). Molecular Cloning: A Laboratory Manual, 3rd edition, Cold Spring Harbor Laboratory Press.
Lab protocols found on microarray labs: [http://www.microarrays.ca/support/proto.html][http://www.daf.jhmi.edu/microarray/protocols.htm] [http://dms.dartmouth.edu/dgml/background/][http://www.hpcgg.org/Microarrays/DNAMicroarray/resources.jsp#A]


[[Category:Molecular biology]]
[[Category:Gene expression]]
[[Category:Bioinformatics]]
[[Category:Glass coating and surface modification]]
[[Category:DNA]]
[[Category:Microarrays]]</body> </html>